tutorials/navigation/navigation_using_navigationpathqueryobjects.rst

```diff
@@ -3,6 +3,14 @@
 Using NavigationPathQueryObjects
 ================================
 
+.. tip::
+
+    Path query parameters expose various options to improve pathfinding performance or lower memory consumptions.
+    
+    They cater to more advanced pathfinding needs that the high-level nodes can not always cover.
+    
+    See the respective option sections below.
+
```

Comment:
This doesn't sound like a tip. It's describing exactly what these objects are for, I assume. Not sure what to do about that.